src/componenents/Form.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { useState } from 'react'
+
+const Form = ({getGames}) => {
+    const [userInput, setUserInput] = useState('placeholder')
+
+    const handleUserInput = (e) => {
+        setUserInput(e.target.value)
+    }
+
+    return (
+            <form onSubmit={(e) => {getGames(e, userInput)}}>
```

Comment:
Semantically, you could add a label here for the form element for accessibility.